lib/command/event/enumerate.php

```diff
@@ -56,13 +56,13 @@ public function __construct( int $limit = 50 ) {
 	 *
 	 * @return int
 	 */
-	public function get_limit(): int {
+	private function get_limit(): int {
 		return $this->limit;
 	}
 
 	/**
 	 * Sets the maximum number of events to return in the response.
-	 *
+	 * @internal
```

Comment:
Should this be internal? Also nitpick, there should be an empty line above `@internal`

Reply:
Yeah, I'm not sure...
I mean, we can define the limit in the constructor, but marking it as `@internal` would mean that we wouldn't want them to use it after instantiating... Which seems weird...

You're right. I'll remove it as well. Thanks 😬